satpy/tests/reader_tests/test_ami_l1b.py

```diff
@@ -194,16 +194,6 @@ def test_get_dataset(self):
             self.assertEqual(val, res.attrs[key])
         self._check_orbital_parameters(res.attrs['orbital_parameters'])
 
-    def test_bad_calibration(self):
-        """Test that asking for a bad calibration fails."""
-        from satpy import DatasetID
-        self.assertRaises(ValueError, self.reader.get_dataset,
-                          DatasetID(name='VI006', calibration='_bad_'),
-                          {'file_key': 'image_pixel_values',
-                           'standard_name': 'toa_outgoing_radiance_per_unit_wavelength',
-                           'units': 'W m-2 um-1 sr-1',
-                           })
-
```

Comment:
Why was this test deleted?

Reply:
Because this is now done at the DataID level, since calibrations levels are enumerations. Hence an invalid calibration with throw an exception there already.